packages/irmin-test/irmin-test.3.6.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Gazagnaire" "Thomas Leonard"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin"
+bug-reports:  "https://github.com/mirage/irmin/issues"
+dev-repo:     "git+https://github.com/mirage/irmin.git"
+doc:          "https://mirage.github.io/irmin/"
+
+build: [
+ ["dune" "subst"] {dev}
+ ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "irmin"        {= version}
+  "ppx_irmin"    {= version}
+  "ocaml"        {>= "4.02.3"}
+  "dune"         {>= "2.9.0"}
+  "alcotest-lwt" {>= "1.5.0"}
+  "mtime"        {>= "1.0.0" & < "2.0.0"}
```

Comment:
Is this upper-bound still expected? mtime.2.0.0 was released 2 months ago

Reply:
Yeah, it's still valid. We haven't updated to the new API. There is a [draft PR](https://github.com/mirage/irmin/pull/2166) that moves the lower bounds to `>= 2.0.0` but would require other dependencies to update as well.